pypesto/ensemble/ensemble.py

```diff
@@ -138,7 +140,8 @@ def condense_to_arrays(self):
         }
 
     def compute_summary(self,
-                        percentiles_list: Sequence[int] = (5, 20, 80, 95)
+                        percentiles_list: Sequence[int] = (5, 20, 80, 95),
+                        weighting: bool = True
```

Comment:
```suggestion
                        weigh: bool = True
```
Option can be `weigh` and values can be `weights`

Reply:
Perhaps this should be `False` by default?

Reply:
is just a matter of taste I think. As `LLH` is not included in a predictor by default, this will get turned to `False` in line 168. The idea behind that for me was that if you include `llh` in predictor for ensembles the default you probably would want is `True`.

Reply:
So in the default setting, `weighting==True` but no weights are used? Might be confusing for users. Could call it something more descriptive, e.g. `weights_if_llh_provided`, or make it `False` by default?